src/collective/mustread/interfaces.py

```diff
@@ -190,10 +192,91 @@ def who_unread(obj, force_deadline=True):
         If ``force_deadline`` is False, returns scheduled users who
         did not read the object at all.
 
+        @guido: i'd suggest to remove force_deadline as we have the get_stats
+        method to get a detailed report which users read and when
+
```

Comment:
This was added as per your spec. I'm neutral, but if you want to remove `force_deadline` here IMO the `deadline` arg should be removed from `schedule_must_read` also. Either this application has a deadline concept and a way to query for that, or else the whole deadline thingy is handled by consuming code based on datetime reporting and this application itself does not care.